kiosk/src/Components/GameMenu.tsx

```diff
@@ -30,6 +28,7 @@ export const GameMenu: React.FC<IProps> = () => {
         <div className="gameMenu">
             {game?.userAdded && !kiosk.locked && (
                 <GenericButton
+                    title={lf("Delete Game")}
```

Comment:
Do you need a label here?

Reply:
In this case button content is provided by the nested svg and other child elements, so we don't want to send a label.